### ISSUE TITLE
attempt to uninstall leads to SE63

### ISSUE THREAD

**Issue by AndreaBorgia-Abo** [abapGit/abapGit#5730]
I had a [zapcommander](https://github.com/tricktresor/zapcommander) installation made when the original [language was German](https://github.com/tricktresor/zapcommander/issues/9).
Pulling the current version with English as main language wouldn't work so I tried uninstalling first, got prompted to fix the translations before being allowed to continue and then this happened:
![immagine](https://user-images.githubusercontent.com/25269970/186489266-ffb0964e-1c74-44ec-bbbb-fe16ad5bc982.png)

Basically, the top menu and the status message at the bottom belong to SE63, which is complaining about untranslated strings.
Exiting the translation and trying to continue uninstalling means loads of prompts to switch language, so I gave up, removed everything via SE80 and reinstalled. No big deal, still the UI was a fun mix.

**Comment by mbtools** [comment]
When a process lands on a dynpro page, the HTML control is not automatically hidden. 

Were you still logged on in English when you tried to uninstall (the repo with original language German)? That should not be allowed by abapGit.

**Comment by AndreaBorgia-Abo** [comment]
Yes, it's the A4H minisap and I only use English. Also, abapgit was just
updated (to give you a reference, not that it's related)

**Comment by mbtools** [comment]
Test with https://github.com/abapGit/abapGit/pull/5731
